release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1188,6 +1188,14 @@ Status displays are paused on all other consoles while the interactive network c
 Previously, the displays could be shown only on a graphical console.
 (link:https://issues.redhat.com/browse/OCPBUGS-19688[OCPBUGS-19688])
 
+[discrete]
+[id="ocp-4-15-mco-mc-ordering"]
+=== MCO gives priority to custom machine config changes
+
+When making changes to your worker nodes through a machine config, the changes are also applied to any node that is in a custom machine config pool. The machine configs are applied in alphanumeric order. As such, previously it was possible that if you changed a parameter for a worker node through a machine config and the same parameter for a custom node at the same time through a separate machine config, the worker machine config could have overridden the custom machine config, based on the order the two were applied. 
```

Comment:
```
based on the order the two were applied. 
```

Seems like it could imply that timing of the object application would make a difference, which is not the case. Perhaps we can phrase it like: `based on the name of the MachineConfig, or in the case of generated MachineConfigs by the controller, the naming of the pool` ?

Reply:
Yes, so when the user provides a kubeletconfig or containerruntimeconfig object, etc., the MCO will generate a MachineConfig based on it, but also put the name of the pool into the generated name.

For example, if you make a `my-kubeletconfig` and target it to worker, it will generate as `99-worker-generated-kubeletconfig` so it doesn't retain any original naming, and this is what used to trip up users the most, since they could have made a `50-my-worker-config` and `51-my-infra-config` directly via MachineConfig to get around the old strict ordering, but they could not work around the generated ones